docs/reports/requirements/1_proj-desc.md

```diff
@@ -62,4 +62,38 @@ Note: the User section will be color-coded to show what's already in place and w
 
 ## PDF Report
 
-I will write my PDF report section here to later be incorporated into the full document.
+### High-level Description (about one paragraph)
```

Comment:
```suggestion
### High-level Description
```
Is the "about one paragraph" portion meant to remain in the final PDF report?

Reply:
I put it there to remind myself, but we don't want to keep it in the final report! Good catch!